utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -52,10 +60,17 @@ class UrlImageParser(
         val textviewWidth = tvContents.width
         val drawableHeight = (drawable as BitmapDrawable).intrinsicHeight
         val drawableWidth = (drawable as BitmapDrawable).intrinsicWidth
-        val newHeight = drawableHeight * textviewWidth / drawableWidth
-        val rect = Rect(0, 0, textviewWidth, newHeight)
-        (drawable as BitmapDrawable).bounds = rect
-        urlDrawable.bounds = rect
+        // To resize the image keeping aspect ratio.
```

Comment:
Is it possible to add a test to verify that these two cases work as expected?

Reply:
Duplicate of this PR is created #205